vowpalwabbit/example_predict.h

```diff
@@ -48,7 +48,8 @@ struct example_predict
   // and each char is a namespace.
   std::vector<std::vector<namespace_index>>* interactions;
 
-  uint32_t _current_reduction_depth;  // Used for debugging reductions.  Keeps track of current reduction level
+  // Used for debugging reductions.  Keeps track of current reduction level.
+  uint32_t _current_reduction_depth;
```

Comment:
it would be nice if the name included the word `debug` so that it is clear

Reply:
Done.